web/src/HUDHeader.scss

```diff
@@ -1,5 +1,7 @@
 @import "constants";
 
+// DELETE THIS
```

Comment:
?

Reply:
Oops! Had a bit more cleanup to do here.